### PR TITLE
Added credscan steps for .NET

### PR DESCRIPTION
**Pull request by sima-zhu**
Here is the aggregate-report with cred scan:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=689974&view=results

### CODE REVIEW

File: eng/pipelines/aggregate-reports.yml [hunk 1]
```diff
@@ -30,6 +30,20 @@ jobs:
       - download: net-core
         artifact: packages
         patterns: '*'
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+        displayName: 'Run CredScan'
+        inputs:
+          toolMajorVersion: V2
+          outputFormat: sarif
+          suppressionsFile: eng/CredScanSuppression.json
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml:38]: Are you planning to add this suppression file in a separate PR?

File: eng/pipelines/aggregate-reports.yml [hunk 2]
```diff
@@ -30,20 +30,24 @@ jobs:
...
         displayName: 'Post Analysis'
         inputs:
-          CredScan: true
+          GdnBreakAllTools: false
+          GdnBreakGdnToolCredScan: true
+          GdnBreakGdnToolCredScanSeverity: Error
+          GdnBreakBaselineFiles: '\\eng\dotnet.gdnbaselines'
+          GdnBreakBaselines: dotnetbaselines
+          GdnBreakSuppressionFiles: '\\eng\CredScanSuppression.json'
+          GdnBreakOutputBaseline: demobaseline
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml]: What is this option mean?

File: eng/pipelines/aggregate-reports.yml [hunk 3]
```diff
@@ -30,20 +30,24 @@ jobs:
...
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@1
+          debugMode: false
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
         displayName: 'Post Analysis'
         inputs:
-          CredScan: true
+          GdnBreakAllTools: false
+          GdnBreakGdnToolCredScan: true
+          GdnBreakGdnToolCredScanSeverity: Error
+          GdnBreakBaselineFiles: '\\eng\dotnet.gdnbaselines'
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml]: Is it easy to create one of these baseline files?

File: eng/pipelines/aggregate-reports.yml [hunk 4]
```diff
@@ -26,10 +26,27 @@ jobs:
     pool:
       vmImage: 'windows-2019'
 
+    timeoutInMinutes: 360
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml:29]: Does it really take 6 hours?

Comment (sima-zhu) [reply eng/pipelines/aggregate-reports.yml:29]: It is currently taking less than 2 hours. I set this for testing.

File: eng/pipelines/aggregate-reports.yml (hunk 3 above)
Comment (sima-zhu) [reply eng/pipelines/aggregate-reports.yml]: I have created the baseline files. However, I have to test how they setup the path. It seems post analysis has trouble to read the file.

**Review by weshaggard — APPROVED** [review]
Please exclude the CredScanSupression.json file.

**Pull request merged**